### DISCUSSION TITLE
Consider renaming the hooks to avoid potential conflicts

### DISCUSSION THREAD

**Discussion by razgraf** [sablier-labs/v2-periphery#112] [Ideas]
The proxy plugin used for the Sablier on-cancel callback is called `onStreamCanceled`.

https://github.com/sablier-labs/v2-periphery/blob/453a35ef662183654138bfe8cab2b523f340fa5b/src/SablierV2ProxyPlugin.sol#L77-L79

Wouldn't renaming it in something more unique (e.g. `onSablierStreamCanceled`) help prevent unintended collisions in the future (e.g. other project using the PRBProxy doing "Potato Streaming")? That's not necessarily intended to improve security (as bad players could still engineer plugins with our original name) but unknowing developers may benefit from this.

**Comment by PaulRBerg** [comment]
This is a good idea, but we've unfortunately run out of time, and we won't be able to implement this if we want to stick to the July 10 deadline :(

Happy to re-consider the idea for V2.x

Also, this is actually a feature request for V2 Core, so I will move the discussion over there.

**Comment by PaulRBerg** [comment]
Closing in favor of https://github.com/sablier-labs/v2-core/discussions/595.
